.gitignore

```diff
@@ -3,3 +3,4 @@ vendor
 phpunit.xml
 coverage
 _build
+.idea/
```

Comment:
This should be defined in your global gitignore. It's not related to this project specific